iterminator/iterminator.py

```diff
@@ -349,9 +333,9 @@ def main():
     if args.dark and args.light:
         error("Don't request both --light and --dark")
     if args.dark:
-        selector.exclude(LIGHT_SCHEMES)
+        selector.filter_light_or_dark(False)
     elif args.light:
-        selector.filter(LIGHT_SCHEMES)
+        selector.filter_light_or_dark(True)
```

Comment:
I don't know why I wrote separate `filter()` and `exclude()` methods -- I think it would have been more natural to have a single `filter()` method taking in a True/False predicate function. I'd still vote for doing that, but it doesn't have to be in this PR.